AUTHORS

```diff
@@ -1,6 +1,7 @@
 AUTHORS
 -------
 
-Corey Nolet - Tetra Concepts LLC
-Keith Turner - Peterson Technologies
-Mike Walch - Peterson Technologies
+Corey Nolet (Tetra Concepts LLC)
+Keith Turner (Peterson Technologies)
+Mike Walch (Peterson Technologies)
+Christopher Tubbs
```

Comment:
This list should be sorted by first or last name.  Conveniently, it was sorted by both before :-)  You can pick the order.